extension/src/shared/layerContainers/floodModel.tsx

```diff
@@ -0,0 +1,146 @@
+import { useTheme } from "@mui/material";
```

Comment:
I separated flood model from building model because this model needs some different appearance.